libexec/python/utils.py

```diff
@@ -172,19 +172,22 @@ def api_get(url,data=None,default_header=True,headers=None,stream=None,return_re
 
     return stream
 
+
 def basic_auth_header(username, password):
     '''basic_auth_header will return a base64 encoded header object to
     generate a token
     :param username: the username
     :param password: the password
     '''
     s = "%s:%s" % (username, password)
+    logger.debug("AUTH: Input username/password: %s", s)
```

Comment:
Err, I don't think that's a good idea. If they have an issue, users will share theirs logs with us and they won't realize they are leaking their DockerHub password at the same time.